app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,59 @@
+class QuestionsController < ApplicationController
+  before_action :set_question, only: %i[ show edit update ]
+  def index
+    @questions = Question.all
+  end
+
+  def show
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def edit
+  end
+
+  def create
+    @question = Question.new(question_params.merge(quiz_id: params[:quiz_id]))
+
+    @answers = params[:question][:answers]
+    @correct_answer = params[:question][:correct_answers][1..-1]
+    @answers_hash = []
```

Comment:
do you need these variables on view or in any other methods of the class? if no, please use local variables